load-tests/tsung/alan-tsung.xml

```diff
@@ -0,0 +1,99 @@
+<tsung loglevel="notice">
+  <clients>
+    <client host="localhost" maxusers="32768" use_controller_vm="true" />
+  </clients>
+
+  <servers>
+	<server host="alan.eba-yps9xqsx.us-west-2.elasticbeanstalk.com" port="80" type="tcp"></server>
+
+  </servers>
+
+  <load>
+    <arrivalphase phase="1" duration="5" unit="second">
+      <users arrivalrate="1" unit="second"></users>
+    </arrivalphase>
+    <arrivalphase phase="2" duration="5" unit="second">
+      <users arrivalrate="2" unit="second"></users>
+    </arrivalphase>
+    <arrivalphase phase="3" duration="5" unit="second">
+      <users arrivalrate="4" unit="second"></users>
+    </arrivalphase>
+    <arrivalphase phase="4" duration="5" unit="second">
+      <users arrivalrate="8" unit="second"></users>
+    </arrivalphase>
+    <arrivalphase phase="5" duration="5" unit="second">
+      <users arrivalrate="16" unit="second"></users>
+    </arrivalphase>
+    <arrivalphase phase="6" duration="5" unit="second">
+      <users arrivalrate="32" unit="second"></users>
+    </arrivalphase>
+    <arrivalphase phase="7" duration="5" unit="second">
+      <users arrivalrate="64" unit="second"></users>
+    </arrivalphase>
+    <arrivalphase phase="8" duration="5" unit="second">
+      <users arrivalrate="128" unit="second"></users>
+    </arrivalphase>
+    <arrivalphase phase="9" duration="5" unit="second">
+      <users arrivalrate="256" unit="second"></users>
+    </arrivalphase>
+    <arrivalphase phase="10" duration="5" unit="second">
+      <users arrivalrate="512" unit="second"></users>
+    </arrivalphase>
+  </load>
+
+  <options>
+    <!-- Set connection timeout to 2 seconds -->
+    <option name="global_ack_timeout" value="2000"></option>
+  </options>
+
+  <sessions>
+    <session name="example" type="ts_http" weight="1">
+      <setdynvars sourcetype="random_string" length="15">
+        <var name="rndstring1" />
+      </setdynvars>
+      <request><http method="GET" url="/"></http></request>
+      <request subst="true">
+        <http content_type="application/json"
+	      method="POST"
+	      url="/api/user"
+		contents='{"user": {"username":"%%_rndstring1%%","password":"password","role":"seller"}}'>
+	</http>
+      </request>
+      <request subst="true">
+	<dyn_variable name="jwt" jsonpath="$.jwt"/>
+        <http content_type="application/json"
+	      method="POST"
+	      url="/api/login"
+		contents='{"user": {"username":"%%_rndstring1%%","password":"password","role":"seller"}}'>
+	</http>
+      </request>
+      <request subst="true">
+        <http content_type="application/json"
+	      method="POST"
+	      url="/api/doughnuts"
+		contents='{"description":"Tsung", "image_url":"none", "name":"Tsung-1","price":10, "quantity":50}'>
+		<http_header name="Authorization" value="Bearer %%_jwt%%"/>
+        </http>
+      </request>
+      <request subst="true">
+        <http content_type="application/json"
+	      method="POST"
+	      url="/api/doughnuts"
```

Comment:
Is there a reason we are defining this route twice?

Reply:
they are separate requests to simulate a Seller workflow of creating multiple donuts